docs/getting-started.md

```diff
@@ -17,6 +17,16 @@ The interactive course with quizzes will guide you through everything you need t
 
 ## Setup
 
+Create a new Next.js app using `create-next-app`:
+
+```bash
+npm init next-app
+# or
+yarn create next-app
+```
+
+## Manual Setup
```

Comment:
I think simply adding `create-next-app` at the top is not sufficient. Right now, we have information that is relevant to both create-next-app setup and manual setup in the following section. If we simply add the instructions for `create-next-app` at the top, it's not obvious that some of the contents inside "Manual Setup" are actually relevant for `create-next-app` users.

I also do think we should add more information regarding `create-next-app` and restructure this page. Can I take care of these changes in a new PR instead of doing it on this PR?

![Artboard](https://user-images.githubusercontent.com/992008/74371504-c6d81980-4d8d-11ea-8453-75c8b8776bd6.png)

Reply:
Updated per comment that `create-next-app` shows some directions:

![Artboard](https://user-images.githubusercontent.com/992008/74372468-85e10480-4d8f-11ea-92c4-328cca1f503f.png)